test/test_helper.rb

```diff
@@ -29,7 +28,7 @@
 
 require 'minitest/reporters'
 Minitest::Reporters.use! [
-  Minitest::Reporters::ProgressReporter.new,
+  !ENV['TIME'] ? Minitest::Reporters::ProgressReporter.new : nil,
```

Comment:
```suggestion
  ENV['TIME'] ? Minitest::Reporters::ProgressReporter.new : nil,
```
I am not sure but it seems to me that the two cases have been reversed

Reply:
nop, it is not correct

Reply:
Which one is not correct ?

Reply:
I let you test the Adeline's suggestion, but it's not the targeted behavior: `ProgressReporter` and `SpecReporter` are exclusive